test/tf/gen_tf_pb.py

```diff
@@ -145,6 +145,14 @@ def biasadd_test(g1):
         tf.nn.bias_add(g1_input, g2_input, name='bias_add1')
 
 
+@tf_test
+def biasadd_scalar_test(g1):
```

Comment:
I think you forgot to commit these new files.

Reply:
added